esp/scm/ws_workunits.ecm

```diff
@@ -1931,9 +1931,136 @@ ESPresponse [exceptions_inline, nil_remove] WUGetNumFileToCopyResponse
     [min_ver("1.69")] int64 Total;
 };
 
+// ----------------------------------------------------------------------------------
+// WUDetails service
+//
+// Request related structures
+ESPstruct WUAttributeFilter
```

Comment:
It may be worth switch Attribute->Property to avoid confusion.

In the layer below properties = attributes + statistics + hints + ???

Reply:
@ghalliday Should the service support filtering by hints? Not sure hint filtering would be useful.

Reply:
Where attribute refers to attributes, statistics and hints, it has been renamed to Property.